packages/node_modules/@ciscospark/react-container-call-incoming/src/index.js

```diff
@@ -0,0 +1,128 @@
+import {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {Howl} from 'howler';
+
+import {
+  callIncoming,
+  listeningForIncomingCalls,
+  removeCallIncoming
+} from '@ciscospark/redux-module-media';
+
+import {
+  NOTIFICATION_TYPE_CALL,
+  createNotification
+} from '@ciscospark/react-container-notifications';
+
+import Ringtone from './media/Ringtone.mp3';
+
+
+export class IncomingCall extends Component {
```

Comment:
Yea, I think I'll split the sounds out into their own thing, then create a module for the sounds.